ava-codemods.js

```diff
@@ -30,8 +26,7 @@ function runScripts(scripts, files) {
 }
 
 const cli = meow(`
-	Usage
-	  $ ava-codemods [<file|glob> ...]
```

Comment:
I prefer how it was. Consistency with other tools is not a goal. They only use that style because that's how Unix tools define it.